lib/assessment.js

```diff
@@ -5,30 +5,67 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  if (arguments.length > 0) {
 
+    var array = arguments['0'];
+    var retMin = array[0];
```

Comment:
Although the previous line will work, this one will not (unless the first element passed into `min` is an array).